sympy/concrete/summations.py

```diff
@@ -790,3 +881,24 @@ def eval_sum_hyper(f, i_a_b):
                     return S.NegativeInfinity
             return None
         return Piecewise(res, (old_sum, True))
+
+
+def _order_growth(expression):
```

Comment:
Why can't you just use `O`? The algorithms in O are going to be much smarter than this.